INSTALL.md

```diff
@@ -5,6 +5,8 @@ System requirements: GalSim currently only supports Linux and Mac OSX.
 
 Table of Contents:
 
+0. [Overall summary](#0-overall-summary)
```

Comment:
Looks like github markdown renumbered the list to start with 1., even though you made a 0.  The headings for the subsections are still 0-6 though.  https://github.com/GalSim-developers/GalSim/blob/%23841/INSTALL.md